wiki/Tournaments/MWC/2023_4K/en.md

```diff
@@ -0,0 +1,200 @@
+4. The mappool is to be played according to the order listed on this page.
+   - Teams may skip playing any beatmap on their second run.
+5. Each team must have 3 members play each map. They may be exchanged freely after a map is concluded.
+6. An optional 3-minute break will be offered between the first and second playthrough of the mappool.
+7. All teams will play their qualifiers in separate rooms. It is *suggested* that teams do not broadcast or share their results publicly to avoid seed manipulation.
+   - Multiple teams may be grouped into a singular room if there are not enough referees available at match time.
+8. Only the top 32 seeded teams will advance to the Round of 32.
+
+### Stage instructions
+
+1. Following the Qualifiers, a double-elimination bracket will be played — for teams in the winners bracket, the winning team continues in the same bracket, while the losing team gets moved to the losers bracket, where any further defeats will eliminate the team from the competition.
+2. Teams in the losers bracket must play up to 2 matches each weekend, starting from the Quarterfinals.
+
+### Win conditions
+
+- In the Qualifiers, teams need to place in the top 32 seeded teams in order to advance to the double-elimination stage.
+- In the Round of 32, teams need to win 5 maps to win a match (best of 9).
+- In the Round of 16 and Quarterfinals, teams need to win 6 maps to win a match (best of 11).
+- In the Semifinals, Finals and Grand Finals, teams need to win 7 maps to win the match (best of 13).
+  - *Note: for the Grand Finals match, the team that is coming from the winners bracket is required to win one "set", that is, one best of 13, to win the tournament. The team coming from the losers bracket, however, needs to win 2 sets.*
+
+### Match instructions
+
+1. A referee will create a multiplayer room 10 minutes before the scheduled match time. Teams must join the lobby in that period.
+   - The room settings are `Game mode: "osu!mania"`, `Team mode: "Team Vs"` and `Score mode: "ScoreV2"`. The room name must follow the pattern of `MWC4K2023: ({Red Team}) VS ({Blue Team})`. The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+2. Each captain will first protect one beatmap from the pool. Captains then proceed to each ban one beatmap from the pool, and they may not ban any of the maps that were protected. The banned beatmaps may not be picked by any team for the entire duration of the match.
+3. Each captain must use `!roll` once in the match's chatroom.
+   - The winner of the `!roll` starts protecting one beatmap, followed by the loser of the `!roll` to protect a beatmap. They may not protect the same beatmap. Protected beatmaps may not be banned.
+   - The winner of the `!roll` starts picking the first beatmap of the match.
+   - The loser of the `!roll` starts banning one beatmap, followed by the winner of the `!roll` to ban a beatmap.
```

Comment:
don't these two rules (2 and 3) say the exact same thing?

Reply:
It does seem a bit redundant